src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -14,12 +15,14 @@ public static class GlobalizationConfigurationExtensions
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
         /// <param name="defaultCulture">Used to set a default culture for the application</param>
+        /// <param name="dateTimeStyles">The <see cref="DateTimeStyles"/> that should be used for date parsing.</param>
         /// <remarks>If defaultCulture not specified the first supported culture is used</remarks>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null, DateTimeStyles? dateTimeStyles = null)

```

Comment:
Done

On 25 May 2016 at 14:29, Andreas Håkansson <notifications@github.com> wrote:

> In src/Nancy/GlobalizationConfigurationExtensions.cs
> <https://github.com/NancyFx/Nancy/pull/2438#discussion_r64571929>:
>
> >          /// <remarks>If defaultCulture not specified the first supported culture is used</remarks>
> > -        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
> > +        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null, DateTimeStyles? dateTimeStyles = null)
>
> This extension should be Globalization and not Cultures to align it with
> what it does
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2438/files/71465f330c14bba292f25ed927eea95ae205bcf7#r64571929>
>